keel-core/src/main/kotlin/com/netflix/spinnaker/keel/verification/VerificationRunner.kt

```diff
@@ -67,7 +89,7 @@ class VerificationRunner(
   }
 
   private val Collection<Pair<*, ConstraintStatus?>>.anyStillRunning: Boolean
-    get() = any { (_, status) -> status == PENDING }
+    get() = !none { (_, status) -> status == PENDING }
```

Comment:
Does `NOT_STARTED` count as 'still running'? i thought it did in some other conditions

Reply:
Yeah, I think so. I fixed this to check for `status?.completed`

Reply:
In this context we're only interested in ones that are actually running -- any that are `NOT_EVALUATED` are the one we're potentially about to launch